src/traces/parcoords-strict/index.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+var index = require('../parcoords/index');
+
+index.plot = require('../parcoords/plot-strict');
+
```

Comment:
Let's duplicate `parcoords/index` content here and then replace `plot` with `plot-strict` in this file.
That way the content of `parcoords/plot.js` won't be included in the bundle.
On another note - wondering can't we move this index into the `parcoords` folder and name it `index-strict` instead?

Reply:
> wondering can't we move this index into the parcoords folder and name it index-strict instead?
A bunch of build stuff relies on traces having their own folders and I didn't want to mess around with that too much

>That way the content of parcoords/plot.js won't be included in the bundle.
plot-strict is just a wrapper around plot after all and plot would need to be in both bundles.